ttrss-config.php

```diff
@@ -89,7 +89,11 @@
 	// *** Feed settings ***
 	// *********************
 
-	define('FORCE_ARTICLE_PURGE', 5);
+        $force_article_purge = 5;
+        if (getenv('FORCE_ARTICLE_PURGE') !== null) {
```

Comment:
Should this be checking against `false`?

https://www.php.net/manual/en/function.getenv.php

Reply:
I'm not sure what PHP will do, if the content of the FORCE_ARTICLE_PURGE environment variable is `0`.

Reply:
If the PHP doc tells the truth, it should return a string containing the character `0`, which should be both `!== null` and `!== false`